addons/website_sale/controllers/main.py

```diff
@@ -458,6 +458,9 @@ def shop(self, page=0, category=None, search='', min_price=0.0, max_price=0.0, t
         ) or ''
 
         values = {
+            'auto_assign_ribbons': lazy(
+                lambda: self.env['product.ribbon'].search([('assign', '!=', 'manual')])
+            ),
```

Comment:
@dere-odoo Your solution was good, but we optimized it a bit by passing it as an args in the method so putting in the controllers felt easier for us